resources/devices/console/nintendo.json

```diff
@@ -7,7 +7,7 @@
             "Device_Name": "New 3DS",
             "Device_Maker": "Nintendo",
             "Device_Brand_Name": "Nintendo",
-            "Device_Pointing_Method": "unknown",
+            "Device_Pointing_Method": "stylus",
```

Comment:
The DS systems all use a stylus for pointing on the lower screen (technically a finger can be used too, but they ship with a stylus).

https://images-na.ssl-images-amazon.com/images/I/51Kw516moFL.jpg